client/src/components/FlashCard/CreateFlashcard.js

```diff
@@ -0,0 +1,109 @@
+import React, {useState} from "react"
+import { Button, Stack, TextField } from "@mui/material"
+import axios from 'axios'
+
+const CreateFlashcard = ({ userId, deckId }) => {
+  console.log(`[CreateFlashcard] deckId is ${deckId}`)
+  const [formValue, setFormValue] = useState({})
+  const [errors, setErrors] = useState({
```

Comment:
After removing the breaking line, your form fields all start in an invalid state and are not evaluated and set to valid unless another field is edited. This is not ideal behavior.

Reply:
I just corrected my form fields. I had extra ' ' which makes them invalid. I also changed my strings to null for simplicity.